app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,17 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp</title>
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
`Ruby on Rails Tutorial Sample App` tìm hiểu I18n nhé
sử dụng  `I18n trong  TOÀN BỘ TEXT của TẤT CẢ CÁC FILE HTML`